src/CommonDropdown.php

```diff
@@ -775,6 +775,11 @@ public function importExternal(
                 $input["name"] = $res_rule["name"];
             }
         }
+        // Merge extra input fields into $input
+        foreach ($external_params as $k => $v) {
+            $input[$k] = $v;
+        }
```

Comment:
To prevent overriding existing data in $input:
```suggestion
        $input += $external_params;
```